package.json

```
@@ -6,6 +6,9 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.2.1",
+    "bootstrap": "^4.6.0",
```

`4.6.0` でハンバーガーメニューが動作しないバグがありますので，4.5.1に変更して下さい。

```suggestion
    "bootstrap": "^4.5.1",
```

```
yarn install --check-files
```

を実行した上でプッシュし直していただけますか。